### PR TITLE
add stop token filter docs #8446

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
add stop token filter docs

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/8446

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@udabhas @varun-lodaya Please provide tech review approval to move this PR forward in the documentation process. Please review this week or provide a peer who can review it. Thank you.

File: _analyzers/token-filters/stop.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Stop
+parent: Token filters
+nav_order: 410
+---
+
+# Stop token filter
+
+The `stop` token filter is used to remove common words (also known as _stopwords_) from a stream of tokens during analysis. Stopwords are typically articles and prepositions such as `a` or `for`. These words do not carry much meaning in search queries and are often excluded to improve search efficiency and relevance.
```
Comment (natebower) [inline _analyzers/token-filters/stop.md:10]:
```suggestion
The `stop` token filter is used to remove common words (also known as _stopwords_) from a token stream during analysis. Stopwords are typically articles and prepositions, such as `a` or `for`. These words are not significantly meaningful in search queries and are often excluded to improve search efficiency and relevance. 
```

File: _analyzers/token-filters/stop.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+title: Stop
+parent: Token filters
+nav_order: 410
+---
+
+# Stop token filter
+
+The `stop` token filter is used to remove common words (also known as _stopwords_) from a stream of tokens during analysis. Stopwords are typically articles and prepositions such as `a` or `for`. These words do not carry much meaning in search queries and are often excluded to improve search efficiency and relevance. 
+
+The default predefined list of English stopwords includes the following words: `a`, `an`, `and`, `are`, `as`, `at`, `be`, `but`, `by`, `for`, `if`, `in`, `into`, `is`, `it`, `no`, `not`, `of`, `on`, `or`, `such`, `that`, `the`, `their`, `then`, `there`, `these`, `they`, `this`, `to`, `was`, `will`, and `with`.
```
Comment (natebower) [inline _analyzers/token-filters/stop.md:12]: Is "default predefined" redundant here?

**Review by natebower — COMMENTED** [review]
@kolchfa-aws One comment and a couple of changes. Thanks!

File: _analyzers/token-filters/stop.md (hunk 2 above)
Comment (kolchfa-aws) [inline _analyzers/token-filters/stop.md:12]:
```suggestion
The default list of English stopwords includes the following words: `a`, `an`, `and`, `are`, `as`, `at`, `be`, `but`, `by`, `for`, `if`, `in`, `into`, `is`, `it`, `no`, `not`, `of`, `on`, `or`, `such`, `that`, `the`, `their`, `then`, `there`, `these`, `they`, `this`, `to`, `was`, `will`, and `with`. 
```

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**